lib/instrumentation/modules/memcached.js

```diff
@@ -34,28 +37,37 @@ module.exports = function (memcached, agent, { version, enabled }) {
 
   // Wrap the generic command that is used to build touch, get, gets etc
   function wrapCommand (original) {
-    return function wrappedCommand () {
-      if (typeof arguments[0] === 'function') {
-        var query = arguments[0]()
-        // If the callback is not a function the user doesn't care about result
-        if (query && typeof query.callback === 'function') {
-          var span = agent.startSpan(`memcached.${query.type}`, 'db', 'memcached', query.type)
-          agent.logger.debug('intercepted call to memcached.prototype.command %o', { id: span && span.id, type: query.type })
-          if (span) {
-            span.setDbContext({ statement: `${query.type} ${query.key}`, type: 'memcached' })
-            const origCallback = query.callback
-            query.callback = agent._instrumentation.bindFunction(function tracedCallback () {
-              span.end()
-              return origCallback.apply(this, arguments)
-            })
-            // Rewrite the query compiler with the wrapped callback
-            arguments[0] = function queryCompiler () {
-              return query
-            }
-          }
-        }
+    return function wrappedCommand (queryCompiler, _server) {
```

Comment:
nit: it doesn't look like the function ever uses `_server`

Reply:
Yah, a "_" prefix on args is my sometimes way of saying "this is the expected position of this argument in the call signature, but I'm not using it". This "_" prefix pattern is this eslint rule (https://github.com/typescript-eslint/typescript-eslint/issues/1510) which I believe is a default in `standard`s set of eslint style rules.

Happy to drop it if you prefer. For trailing args it is purely for self-documenting. For non-trailing args (e.g. https://github.com/elastic/apm-agent-nodejs/blob/3928f8d88d30eb9a012f4b2b5bd9600be2bbc19a/lib/noop-transport.js#L28) it is more helpful/necessary.